setup.cfg

```diff
@@ -16,6 +16,7 @@ max-line-length = 120
 ignore =
         F403,
         F405,
+        F811,
```

Comment:
https://lintlyci.github.io/Flake8Rules/rules/F811.html

why?

Reply:
Pytest fixtures. If you import a fixture by name, and then use it as you must (by including it as an argument to the test) then python thinks you're redefining something that you're actually just calling.

Reply:
It was either ignore F811 or add #noqa to 10 lines

Reply:
Or use a * import.

Reply:
Ignoring `F811` means that in any file you can import twice the same library... I know that is a nightmare to add `# noqa` to 10 lines but it's also a bad practice to allow to duplicate imports. We need to find a balance between these two cases.

Reply:
Compromise on a * import then? That seems to be the common practice with pytest

Reply:
Whatever you prefer most!